app/config/sources.inc.php

```diff
@@ -316,8 +321,10 @@
         'supported_locales' => $mozillaorg,
     ],
     'firefox/sendto.lang' => [
-        'priority'          => 1,
+        'flags'             => [
+          'obsolete'    => ['all'],
```

Comment:
Once more: this is NOT an obsolete file.

Also, these arrays should look like:

```
'mozorg/plugincheck-redesign.lang' => [
    'flags'             => [
        'obsolete' => ['all'],
    'supported_locales' => $mozillaorg,
    ],
],
```